features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("the {string} should be visible") do |selector|
+  expect(page).to have_selector :css, "##{selector}"
+end
+
+When("I click on {string} link") do |link|
+  click_link_or_button link
+end
+
+Then("I should be redirected to landing page") do
```

Comment:
Read this out loud. I think there is a missing `the` somewhere... But then again, English is not my native language...

Reply:
👍